client/components/jetpack/card/jetpack-product-card/upgrade-nudge.tsx

```diff
@@ -0,0 +1,144 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { translate as globalTranslate, useTranslate, TranslateResult } from 'i18n-calypso';
+import { isFinite } from 'lodash';
+import React, { useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import Gridicon from 'components/gridicon';
+import { preventWidows } from 'lib/formatting';
+import PlanPrice from 'my-sites/plan-price';
+import { JETPACK_OFFER_RESET_UPGRADE_NUDGE_DISMISS } from 'my-sites/plans-v2/constants';
+import { savePreference } from 'state/preferences/actions';
+import { getPreference } from 'state/preferences/selectors';
+import { getPlan } from 'lib/plans';
+
+/**
+ * Type dependencies
+ */
+import { JetpackRealtimePlan } from 'lib/plans/types';
+
+// TODO: replace this hardcoded data with real data when we have an endpoint for it or
+// a more definitive place in our own codebase.
+const DEFAULT_FEATURES = [
+	{
+		title: globalTranslate( 'Real-Time Backup' ),
+		subtitle: globalTranslate( 'Every edit gets saved immediatelly' ),
+	},
+	{
+		title: globalTranslate( 'Real-Time Scan' ),
+		subtitle: globalTranslate( 'On-demand scanning' ),
+	},
+];
```

Comment:
The code should not use `translate` if it's placeholder text, as committing this will trigger the translation of these strings. Honestly, I would prefer we simply declare a type here:
```
type Feature = {
    title: string;
    subtitle: string;
}
```
and then use `Feature[]` below. Test data like this should live in the fixtures file, not in the component itself.

Reply:
You're right. We shouldn't translate placeholder text.